### ISSUE TITLE
How do I output all predictions on validation set?

### ISSUE THREAD

**Issue by jensonwang99** [jdb78/pytorch-forecasting#1284]
After I trained a model, how can I use it to predict on all validation dataset, instead of only on last MAX_PREDICTION_LENGTH time steps?
